vips/vip-408/bsctestnet.ts

```diff
@@ -0,0 +1,188 @@
+
+export const vip408 = () => {
+  const meta = {
+    version: "v2",
+    title: "VIP-408 [Sepolia] New Ethena pool (Part-II)",
+    description: `#### Summary`,
+    forDescription: "Process to configure and launch the new pool",
+    againstDescription: "Defer configuration and launch of the new pool",
+    abstainDescription: "No opinion on the matter",
+  };
+  return makeProposal(
+    [
+      {
+        target: sepolia.VTREASURY,
+        signature: "withdrawTreasuryToken(address,uint256,address)",
+        params: [MocksUSDe, parseUnits("10653", 18), sepolia.NORMAL_TIMELOCK],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: MocksUSDe,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: MocksUSDe,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, parseUnits("10653", 18)],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: VsUSDe_Ethena,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["7200"],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VsUSDe_Ethena,
+            parseUnits("0.9", 18),
+            parseUnits("0.92", 18),
+            parseUnits("10653", 18),
+            sepolia.VTREASURY,
+            parseUnits("50000000", 18),
+            parseUnits("0", 18),
+          ],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.VTREASURY,
+        signature: "withdrawTreasuryToken(address,uint256,address)",
+        params: [MockUSDC, parseUnits("10000", 6), sepolia.NORMAL_TIMELOCK],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: MockUSDC,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, 0],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: MockUSDC,
+        signature: "approve(address,uint256)",
+        params: [sepolia.POOL_REGISTRY, parseUnits("10000", 6)],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: VUSDC_Ethena,
+        signature: "setReduceReservesBlockDelta(uint256)",
+        params: ["7200"],
+        dstChainId: LzChainId.sepolia,
+      },
+      {
+        target: sepolia.POOL_REGISTRY,
+        signature: "addMarket((address,uint256,uint256,uint256,address,uint256,uint256))",
+        params: [
+          [
+            VUSDC_Ethena,
+            parseUnits("0", 18),
+            parseUnits("0", 18),
+            parseUnits("10000", 6),
+            sepolia.VTREASURY,
+            parseUnits("50000000", 6),
+            parseUnits("46000000", 6),
+          ],
+        ],
+        dstChainId: LzChainId.sepolia,
+      },
+
+      {
```

Comment:
I would move to the first VIP the command to pause the PT markets. We could move to this VIP the command to set the protocol seize percentage for USDC and sUSDe, if we need it, due to the constraints on the number of commands per VIP

Reply:
The idea being this comment is to have as many commands as possible related to each market together

Reply:
[8d26d1c](https://github.com/VenusProtocol/vips/pull/440/commits/8d26d1ca0af577598cd52b2955c9d065202be5f7)